### ISSUE TITLE
Is this project active?

### ISSUE THREAD

**Issue by fj-sanchez** [micro-os-plus/cmsis-plus#51]
Hi,

I just found about this project but the last commit seems to be from 2018, is this project still active?
@ilg-ul I see that you still answer the issues, but is any more development expected for this project?

Thank you in advance for the amazing work done here.

**Comment by ilg-ul** [comment]
Definitely, there is more development expected.

I'm currently working on the tools I plan to use to run the tests, then the project will be split into separate packages, each with its tests, and brought together automatically by xpm, or as github submodules.

So yes, this project is still alive, just that those who use it did not have any issue, so there were no changes.

**Comment by fj-sanchez** [comment]
That was a quick answer!! Definitely encouraging to try out. I would like to play a bit with the project, I just got an STM-32 dev board (NUCLEO-F413ZH) which I think would be suitable, where should I start? I'm a bit lost right now... I'm currently trying out STM32CubeIDE but I'm happy to stick to something else if that ease the process of getting started.

**Comment by ilg-ul** [comment]
Yes, I know, as it is now organised, the project is a bit difficult to understand, and this is the reason I'm working on some tools to automate project build and testing.

I suggest you take a look at the demo projects:

https://github.com/micro-os-plus/eclipse-demo-projects

and the test projects

https://github.com/micro-os-plus/eclipse-test-projects

As a general idea, there will be a template project similar to the SiFive Template (https://github.com/micro-os-plus/sifive-templates-xpack) that will generate projects, with the option to use an RTOS or not, and the RTOS choices will be FreeRTOS and µOS++.

**Comment by ilg-ul** [comment]
BTW, even if you are not interested in RISC-V, I'd be glad to know your opinion on the approach used in that template project.

Please post your comments in the project forum (https://www.tapatalk.com/groups/xpack/general-os-discussions-f22/)
